tests/op_analytics/cli/pulls/defillama/test_defillama.py

```diff
@@ -1,149 +1,244 @@
-            "bridged_to": 18875986.0,
-            "minted": None,
-            "unreleased": None,
-            "id": "1",
-            "name": "Tether",
-            "symbol": "USDT",
-        },
-        {
-            "chain": "Optimism",
-            "dt": "2022-05-19",
-            "circulating": 19207042.0,
-            "bridged_to": 19207042.0,
-            "minted": None,
-            "unreleased": None,
-            "id": "1",
-            "name": "Tether",
-            "symbol": "USDT",
-        },
-        {
-            "chain": "Optimism",
-            "dt": "2022-05-20",
-            "circulating": 19358232.0,
-            "bridged_to": 19358232.0,
-            "minted": None,
-            "unreleased": None,
-            "id": "1",
-            "name": "Tether",
-            "symbol": "USDT",
-        },
-        {
-            "chain": "Optimism",
-            "dt": "2022-05-21",
-            "circulating": 19368126.0,
-            "bridged_to": 19368126.0,
-            "minted": None,
-            "unreleased": None,
-            "id": "1",
-            "name": "Tether",
-            "symbol": "USDT",
-        },
+# Sample data resembling the API response
+sample_summary = {
+    "peggedAssets": [
+        {"id": "sample-stablecoin", "name": "Sample Stablecoin"},
+        {"id": "another-stablecoin", "name": "Another Stablecoin"},
+    ],
+    "pegType": "peggedUSD",
+}
+
+sample_breakdown_data = {
+    "pegType": "peggedUSD",
+    "chainBalances": {
+        "Ethereum": {
+            "tokens": [
+                {
+                    "date": current_timestamp,
+                    "circulating": {"peggedUSD": 1000000},
+                    "bridgedTo": {"peggedUSD": 200000},
+                    "minted": {"peggedUSD": 1200000},
+                    "unreleased": {"peggedUSD": 0},
+                },
+                {
+                    "date": current_timestamp - 86400 * 2,  # 2 days ago
+                    "circulating": {"peggedUSD": 900000},
+                    "bridgedTo": {"peggedUSD": 180000},
+                    "minted": {"peggedUSD": 1080000},
+                    "unreleased": {"peggedUSD": 0},
+                },
+            ]
+        }
+    },
+    "id": "sample-stablecoin",
+    "name": "Sample Stablecoin",
+    "address": "0xSampleAddress",
+    "symbol": "SSC",
+    "url": "https://samplestablecoin.com",
+    "pegMechanism": "fiat-backed",
+    "description": "A sample stablecoin for testing.",
+    "mintRedeemDescription": "Mint and redeem instructions.",
+    "onCoinGecko": True,
+    "gecko_id": "sample-stablecoin",
+    "cmcId": "12345",
+    "priceSource": "coingecko",
+    "twitter": "@samplestablecoin",
+    "price": 1.00,
+}
+
+another_sample_breakdown_data = {
+    # Similar structure with required fields
+    "pegType": "peggedUSD",
+    "chainBalances": {
+        "Binance Smart Chain": {
+            "tokens": [
+                {
+                    "date": current_timestamp,
```

Comment:
I would advise against more complex test data like this. Instead maybe the function you are testing should accept cutoff_date directly rather than accepting the timedelta from the present.  That way you can keep your test data simpler. Ideally test data would be straight up copied from a real API response without any modifications.